### DISCUSSION TITLE
Is it appropriate for the save method to return a Ruby Array?

### DISCUSSION THREAD

**Discussion by kojix2** [heronshoes/red_amber#156] [Ideas]
https://github.com/heronshoes/red_amber/blob/e419aa709fcbc6dd7d981426b36124f861d5fe93/lib/red_amber/data_frame_loadsave.rb#L21

I feel a little uneasy with the save method returning an Array. In particular, when using RedAmber in IRB, a large array is displayed in the terminal when the save method is executed. This is because RedArrow::Table returns a Ruby Array when the save method is executed. Therefore, this may not necessarily be a problem with RedAmber, but rather with RedArrow::Table. It is possible that the specification of the save method returning an Array is valid in some cases, such as when using method chaining. However, in those cases, it is possible that RedArrow::Table should be returned instead of a Ruby Array.

**Comment by heronshoes** [comment]
I agree with you. It is easy to return another value, such as filename or file size (like IO::write)?

But I will also request change in Red Arrow. It should be fix in Red Arrow.
